src/main/java/hudson/plugins/jira/Updater.java

```diff
@@ -348,7 +348,7 @@ protected void appendAffectedFilesToDescription(Entry change, StringBuilder desc
 
     protected void appendChangeTimestampToDescription(StringBuilder description, JiraSite site, long timestamp) {
         DateFormat df = null;
-        if (!StringUtils.isBlank(site.getDateTimePattern())) {
```

Comment:
haha i meant not to move negation to the other place but get rid of it:
```
if (StringUtils.isBlank(site.getDateTimePattern())) {
  // default format for current locale
df = new SimpleDateFormat("d/M/yy hh:mm a", Locale.getDefault());
} else {
  df = new SimpleDateFormat(site.getDateTimePattern());
}
```

Reply:
arff :)

Reply:
should be ok now :)